src/entities/abTrail/lib/utils/calculation.ts

```diff
@@ -40,7 +38,28 @@ export const getDistance = (from: Point, to: Point): number => {
 
 const MaxDistanceDiff = 10000;
 
-export const getEquidistantPoint = (errorPath: SkPath): Point | null => {
+interface IPath {
```

Comment:
I wonder if it's necessary to create an Interface that is not going to be exported and used anywhere else

Reply:
no need, removed

Reply:
Yes, otherwise external code will highlight error re absence of 
private points: Point[];
This is internal thing of UtilsPath, but inetrface interface IUtilsPath doesn't share this property, so no error.